POLICIES.md

```diff
@@ -132,4 +137,32 @@ spec:
 ```
 where `storeSecret` identifies an in scope Kubernetes secret which contains `username` and `password` data items which are used to authenticate with the server referenced in `storeURL`.
 
+### vulnerability
+
+Vulnerability policies enable you to admit or deny pod admission based on the security status of the container images within the pod. Vulnerability-based admission is available for:
+* Vulnerability Advisor for IBM Cloud Container Registry
```

Comment:
What happens if you enable IBMVA for an image that doesn't support it? Should we have a line saying what counts as a supported image for VA and what happens if you put IBMVA on an image that isn't in the IBM registry?